tests/gnocchi.sh

```diff
@@ -40,4 +60,9 @@ for daemon in "${DAEMONS[@]}"; do
     fi
 done
 
+gnocchi resource list | grep generic
+
+gnocchi metric create --archive-policy-name high
+gnocchi metric list | grep high
+
```

Comment:
This file should go away, correct, as you've factored into chunks below?

Reply:
Yes! oops.